msgs/action/PlanPickPlace.action

```diff
@@ -1,7 +1,23 @@
 # goal
+
+# planning group properties
+string arm_group_name
+string hand_group_name
+
+# robot model and links
+string eef_name
+string hand_frame
+
+# object and surfaces
 string object_id
-string end_effector
+string[] support_surfaces
+
+# picking
+string grasp_provider_plugin_name
+moveit_msgs/Grasp grasp
+geometry_msgs/Pose grasp_frame_transform
```

Comment:
What does this mean? Please add all the explanation to this `.action` file, it's the main piece of documentation for the user. Make sure it has all the details and doesn't require detective work.